docs/document/website/versioned_docs/version-1.12.0/components/desktop/tooltip.md

```diff
@@ -0,0 +1,97 @@
+---
+id: version-1.12.0-tooltip
+title: Tooltip
+sidebar_label: Tooltip
+original_id: tooltip
+---
+
+## Overview
+
+Tooltip component allows the user to display a short explanation of the target element when hovering or focusing on the element.
+
+<div class="sample-container" id="tooltip">
+  <div id="sample-container__components"></div>
+</div>
+<script src="/js/samples/desktop/tooltip.js"></script>
+
+---
+
+## Specification
+
+### Property
+
+Here is a list of properties that can be used for modifying the component:
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| className | string | ""  | Component class name | |
+| id | string | ""  | Component id name | |
+| placement | string | top  | The position of the component relative to the target element | Available options: `top`, `left`, `right`, `bottom` |
+| title | string | ""  | The text shown in the component | |
+| container *1 | string/HTMLElement | "" | The target element to display the component | The `title` value will be displayed in the situation below<ul><li>When the container element is hovered</li><li>When the container element is focused</li></ul> |
+| describeChild *2 | boolean | false  | Provide additional information for assistive technologies to describe the contents of the Tooltip | It is used to control how the Tooltip interacts with the child element and provides additional accessibility information|
+
+> *1: Please use **focusable elements** as much as possible and avoid abusing unfocusable elements for accessibility.
+
+> *2: By understanding how to use the describeChild property, you can effectively improve the accessibility and usability of your application.
+> - Set `describeChild` to `false` if the Tooltip represents the purpose of the element itself.
+>   - If the Tooltip is used to provide a brief explanation or label for the element it is attached to, you can set `describeChild` to `false`.
+>   - The `aria-label` attribute will be added to the `button` element with the value of the `title` property.
```

Comment:
To explain the general rule not an example, it should be like this.
If using the example, it will be better to add this sentence under the "For example..." sentence.

```suggestion
>   - The `aria-label` attribute will be added to the first child element of the Tooltip component with the value of the `title` property.
```